R/kelp.R

```diff
@@ -0,0 +1,110 @@
+#' Create a kelp object for interacting with SeaweedFS.
+#'
+#' This is a slightly higher level abstraction than [seaweed_master()] and
+#' [seaweed_volume()].
+#'
+#' @keywords export
```

Comment:
```suggestion
#' @export
```

currently not done - make this change then redocument to see the object appear in the NAMESPACE file